webpack/envs/baseConfig.js

```diff
@@ -13,7 +14,7 @@ const {
 exports.baseConfig = {
   mode: NODE_ENV,
   devtool: "cheap-module-source-map",
-  stats: "errors-only",
+  stats: "normal", // or, `errors-only`
```

Comment:
Lets keep this as normal. It slows things down a bit, but it's needed to trace dependencies.